hardware/computemodule/cmio-camera.md

```diff
@@ -11,7 +11,7 @@
 ## Quickstart
 
 1. On the compute module, run `sudo raspi-config` and enable the camera.
-1. Next, run `sudo wget http://goo.gl/U4t12b -O /boot/dt-blob.bin`
+1. Next, run `sudo wget https://goo.gl/U4t12b -O /boot/dt-blob.bin`
```

Comment:
@JamesH65 Should we be using rpf.io for short-links instead of goo.gl ? :shrug:

Reply:
Or be transparent with binary URLs and use the full target link?

Reply:
Why use short links at all?